src/scripts/treelite_python/score.py

```diff
@@ -118,19 +112,7 @@ def run(args, unknown_args=[]):
     logger.info(f"Loading data for inferencing")
     with metrics_logger.log_time_block("time_data_loading"):
```

Comment:
could you move the line 139 (previous line number)
`dmat = treelite_runtime.DMatrix(my_data)`

 in this block? because it's kinda part of data prep?

Reply:
not sure which block you intend this to be moved to? It is in the block time_data_loading